### PR TITLE
Added british english version of words, i.e. zucchini

### PR DESCRIPTION
**Pull request by JoelBalmer**
I'm a big fan of Dr Greger and a huge consumer of nutritionfacts videos, podcast, etc. This is one way I thought I could contribute to the cause!

Please comment if you have any thoughts :) thanks!

### CODE REVIEW

File: app/src/main/res/values-en-rGB/strings.xml [hunk 1]
```diff
@@ -0,0 +1,894 @@
+<resources>
```
Comment (caugner) [inline app/src/main/res/values-en-rGB/strings.xml]: Shouldn't the directory be named `values-en-GB` rather than `values-en-rGB`?

Comment (JoelBalmer) [reply app/src/main/res/values-en-rGB/strings.xml]: @caugner I can't remember the exact research I did at the time, but I tested it out and I worked, and this I found just now seems to validate my previous thinking:

https://stackoverflow.com/a/23885951/2501304

What do you reckon?

Comment (caugner) [reply app/src/main/res/values-en-rGB/strings.xml]: Sorry, you're absolutely right. I'll correct my review.

**Comment by caugner** [comment]
It might be worth waiting for the code of the current v15 version, because it adds several strings related to "How not to diet".

**Comment by slavick** [comment]
Thank you for the PR! NutritionFacts.org has partnered with official translators going forward. If you would like to apply to be an official translator, you can do that [here](https://nutritionfacts.org/volunteer/).

**Pull request closed without merging**